Model/Attribute/Source/CustomerExemptionType.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Taxjar_SalesTax
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Taxjar
+ * @package    Taxjar_SalesTax
+ * @copyright  Copyright (c) 2017 TaxJar. TaxJar is a trademark of TPS Unlimited, Inc. (http://www.taxjar.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+
+namespace Taxjar\SalesTax\Model\Attribute\Source;
+
+class CustomerExemptionType extends \Magento\Eav\Model\Entity\Attribute\Source\AbstractSource
+{
+    public const WHOLESALE = 'wholesale';
```

Comment:
@ryanreeves-taxjar On PHP 7.0.x you'll get the following error for specifying the visibility of class constants:

```
Parse error: syntax error, unexpected 'const' (T_CONST), expecting variable (T_VARIABLE)
```

Support for this was introduced in PHP 7.1. Magento 2 [allows PHP 7.0.x up to version 2.2](https://devdocs.magento.com/guides/v2.2/install-gde/system-requirements-tech.html).